### DISCUSSION TITLE
Using Fanout Exchange but consumer not yet ready

### DISCUSSION THREAD

**Discussion by Kartikey-star** [rabbitmq/amqp091-go#284] [Q&A]
I am creating a fanout exchange and publishing messages to it. There is no queue bound to the exchange yet. Is it possible to route the already published  messages to the queue once it is created with the binding to the exchange.

**Comment by Zerpet** [comment]
> Is it possible to route the already published messages to the queue once it is created with the binding to the exchange.

Short answer: no 🙂  A potential solution is that your publisher declares the queue and the bindings before publishing. If the consumer declares again the queue and bindings, it will be a no-op.

**Reply by Kartikey-star** [reply]
Thanks a lot @Zerpet .
